Ubuntu/README.md

```diff
@@ -48,9 +48,10 @@ sudo nano /etc/rc.local
 ```
 - Add command to the rc.local file
 ```bash
-cd /home/(User name)/U6143_SSD1306/C 
-sudo make clean 
-sudo make 
+#!/bin/sh
+cd /home/(User name)/U6143_SSD1306/C
```

Comment:
Case sensitivity mark.  Should be U6143_ssd1306

Reply:
When it installs, the case is as I listed. Threw me for a bit when I was initially trying to get it working.

Reply:
The way it installs is based on how you clone the repo, if you clone https://github.com/UCTRONICS/U6143_SSD1306.git it will clone into the all-caps directory, but the way that the repo is configured, it's actual intended repo name is U6143_ssd1306.

Reply:
Fully tracking that... But look at how the case is in the Ubuntu fork. The clone line case is not the same as the compile lines.

Simply making it easier for those who dont catch that.

Reply:
No doubt.  Just sharing what my own experience was in setting this up.  This probably isn't the right PR to comment on this inconsistency on but I'm still pretty fresh in actually interacting in github.  Appreciate the response.

Reply:
Changed the casing in subsequent commit, happy either way based on what's most clear